src/java.rmi/share/classes/java/rmi/server/RemoteObject.java

```diff
@@ -357,7 +359,11 @@ public String toString() {
      * <code>"UnicastServerRef2"</code>, no data is written by the
      * <code>writeExternal</code> method or read by the
      * <code>readExternal</code> method.
+     *
+     * @param  out the {@code ObjectOutputStream} to which data is written
+     * @throws IOException if an I/O error occurs
      */
+    @java.io.Serial
     private void writeObject(java.io.ObjectOutputStream out)
         throws java.io.IOException
```

Comment:
It's a little odd to see an import of java.io.IOException (which is used by the `@throws` tag) but not to have it used here. There's another occurrence down below. These files have a mixture of imports and fully qualified names, which I find irritating, which might be nice to clean up. But you might find beyond the scope of this change. So, clean up as much of this as you like, or none if you prefer, no big deal.